litmus-portal/graphql-server/graph/schema.graphqls

```diff
@@ -173,7 +197,10 @@ type Query{
 
   users: [User!]! @authorized
 
+  # [Deprecated soon]
   getScheduledWorkflows(project_id: String!): [ScheduledWorkflows]! @authorized
+
+  getWorkflows(project_id: String!): [Workflows]! @authorized
```

Comment:
I think @ishangupta-ds needed to select only a few workflows and not all workflows, can we add another parameter - an array of string that will have the workflow-ids. If it is empty we send all else only those specific workflows

Reply:
Purpose of the API is to deprecate `getScheduledWorkflows` and `getWorkFlowRuns`. and if he wants workflows by ID. we can make another mutation getWorkflowByID

Reply:
There will be two mutation for workflow (List-->getWorkflows, Get --> getWorkflowByID )

Reply:
We can change the naming convention to getWorkflow to Listworkflow

Reply:
We can have the same mutation do both right? that reduces extra queries, the change will not be too much, the array could be kept optional. That way the frontend can only send the projectID and get all the workflows or send an array also to get specific workflows.

Reply:
Yes, But every API should have a single purpose. I have added a new Query to get workflow by ID

Reply:
Just like CRUDL

Reply:
Can you change it to an array of ids, if we want more than 1 we have to perform multiple queries

Reply:
If they want an array of workflow ids they will call `ListWorkflow`. If single workflow, call `GetWorkflow`

Reply:
I meant array of specific workflows, ListWorkflows will give all the data. I think that was the main problem that @ishangupta-ds had.

Reply:
yeah @rajdas98 ... @gdsoumya is right about the requirement, we need the ID selector too, this API too solves the issue but the thing is we can have a single API if user gives an array of ID send data for only those workflow schedules i.e. workflow ids and if the array is empty send all workflow data in mongoDB. If you planning to make 2 APIs then okay but if just one then it requires a ID filter which will be an array to filter out just the data of workflows that we need for frontend.